server/schedule/filter/filters.go

```diff
@@ -583,6 +583,7 @@ func (f *specialUseFilter) Target(opt opt.Options, store *core.StoreInfo) bool {
 const (
 	specialUseKey       = "specialUse"
 	specialUseHotRegion = "hotRegion"
+	specialUseReserved  = "reserved"
 )
 
-var allSpecialUses = []string{specialUseHotRegion}
+var allSpecialUses = []string{specialUseHotRegion, specialUseReserved}
```

Comment:
Can  `hot region` scheduler balance to the reserved store?

Reply:
SpecialUseFilter is added in `balanceSolver`, then `hot region` scheduler can only balance to stores for hot region.